read_zdf_vis_3d.py

```diff
@@ -1,38 +1,32 @@
-"""
-Import ZDF point cloud and visualize it.
-"""
+"""Import ZDF point cloud and visualize it."""
 
+import math
 import numpy as np
 import matplotlib.pyplot as plt
-from vtk_visualizer import plotxyzrgb
 import zivid
+import pptk
 
 
-def _main():
-
-    app = zivid.Application()
+def _display_rgb(rgb):
+    """Create and display depthmap.
 
-    # The Zivid3D.zdf file has to be in the same folder as this sample script.
-    filename_zdf = "Zivid3D.zdf"
+    Args:
+        rgb: RGB matrix
```

Comment:
I would use RGB image for consistency with other samples. See also the comment for line 27.